nemo_skills/training/nemo_rl/start_grpo.py

```
@@ -315,18 +317,37 @@ def main() -> None:
+            nemo_gym_actor,
+            cluster,
+            dataloader,
+            val_dataloader,
+            loss_fn,
+            logger,
+            checkpointer,
+            grpo_state,
+            master_config,
+        ) = setup_result
+    else:
+        raise ValueError(f"Expected 9 or 10 elements in setup_result, got {len(setup_result)}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)